lib/log.ex

```diff
@@ -0,0 +1,52 @@
+defmodule KvStorageRedis.Log do
+  @levels %{
+    emergency: 8,
+    alert: 7,
+    critical: 6,
+    error: 5,
+    warning: 4,
+    notice: 3,
+    info: 2,
+    debug: 1
+  }
+
+  @conf_level get_in(Application.get_env(:kv_storage_redis, __MODULE__), [:level])
+  @conf_priority get_in(@levels, [@conf_level])
+
+  @spec level :: atom()
```

Comment:
Лениться не нужно, даже в таких игрушечных проектах.
```suggestion
  @type level :: :emergency | :alert | :critical | :error | :warning | :notice | :info | :debug
  @spec level :: level()
```
Но неужели вы думаете, что этот тип впервые в мире потребовался именно вам?  :)
https://hexdocs.pm/logger/Logger.html#t:level/0

Подглядывать можно.